### PR TITLE
fixed tooltip not overlaying

### LINKED ISSUE #2113: Tooltip is not overlaying
The tooltip should overlay (have a higher z-index) than the floating-wrapper element. This is for some reason not the case despite the relevant tool-tip element having a z-index of 2000 ...

grafik

### PR DESCRIPTION
**Pull request by PaarthAgarwal**
Fixes #2113.
The issue arose because of the opacity property in .button:hover. I've replaced opacity with rgba while keeping UI the same as before. 
Before the fix
Screenshot from 2023-02-01 04-29-29

After the fix
Screenshot from 2023-02-01 04-25-30

### CODE REVIEW

**Review by moneymanolis — CHANGES_REQUESTED** [review]
Has to be resolved differently.

File: src/cryptoadvance/specter/static/style.css [hunk 1]
```diff
@@ -152,7 +152,11 @@
    */
 
   .button {
-    @apply h-11 rounded-xl text-lg flex items-center space-x-2 justify-center bg-dark-600 hover:opacity-80 px-10 cursor-pointer w-[calc(688px/2)] mb-3;
+    @apply h-11 rounded-xl text-lg flex items-center space-x-2 justify-center bg-dark-600 hover:opacity-80 px-10 cursor-pointer w-[calc(688px/2)] mb-3 z-10;
```
Comment (moneymanolis) [inline src/cryptoadvance/specter/static/style.css:155]: For some reason, `z-0` doesn't work.

**Comment by moneymanolis** [comment]
@otkstdio I made an attempt in https://github.com/cryptoadvance/specter-desktop/pull/2116/commits/c3fa48455b0ec13d6aa75bd39c5275d353dbba81, let me know what you think about it.

**Comment by otkstdio** [comment]
> 

Did the attempt work?

**Comment by moneymanolis** [comment]
@otkstdio I've rebuilt the `output.css `- now it's fine, merging ... @PaarthAgarwal thanks, please keep in my mind for the next PRs to not change the output.css directly. See also comment above.

**Comment by PaarthAgarwal** [comment]
> @otkstdio I've rebuilt the `output.css `- now it's fine, merging once the tests are green. @PaarthAgarwal thanks, please keep in my mind for the next PRs to not change the output.css directly. See also comment above.

If I make changes in style.css will they be automatically reflected in output.css? If not then how to build output.css again from style.css?

**Review by moneymanolis — APPROVED** [review]

**Comment by PaarthAgarwal** [comment]
> @PaarthAgarwal You need to run `npm run tailwind` for the building of `output.css`. @otkstdio we need to put all this in the developer documentation for the front end.

Oh I see
Should I open a PR to add this in docs?

**Pull request merged**